tmt/steps/provision/local.py

```diff
@@ -133,6 +133,34 @@ def pull(
             extend_options: Optional[list[str]] = None) -> None:
         """ Nothing to be done to pull workdir """
 
+    def acquire_log(self, log_name: str) -> Optional[str]:
+        """fetch and return content of a requested log"""
+        if log_name == 'dmesg':
+            log_content = self.execute(Command('dmesg')).stdout
+        if log_content:
```

Comment:
If `log_name` is not `dmesg`, then at this moment, Python will blow up because `log_content` would be an unknown name. That's a bug, but we can avoid it by refactoring the code: do we even have to store the content? I'd say it's enough to return it right away:

```python
        if log_name == 'dmesg':
            return self.execute(Command('dmesg')).stdout
        return None
```

Reply:
Thanks for your review^^,gonna to modify the code accordingly.